packages/stryker-javascript-mutator/test/helpers/initSinon.ts

```diff
@@ -1,5 +1,5 @@
 import * as sinon from 'sinon';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here, do we need all of the logging or just one part?

Reply:
Good point. Renamed